demo/pom.xml

```diff
@@ -0,0 +1,111 @@
+                                <configuration>
+                                        <imageName>curd-api</imageName>
+                                        <baseImage>openjdk:8</baseImage>
+                                        <entryPoint>["java", "-jar", "/${project.build.finalName}.jar"]</entryPoint>
+                                        <!-- copy the service's jar file from target into the root directory of the image -->
+                                        <resources>
+                                                <resource>
+                                                        <targetPath>/</targetPath>
+                                                        <directory>${project.build.directory}</directory>
+                                                        <include>${project.build.finalName}.jar</include>
+                                                </resource>
+                                        </resources>
+                                </configuration>
+                        </plugin>
+
+                </plugins>
+
+        </build>
+
+        <dependencies>
+                <dependency>
+                        <artifactId>spring-boot-starter-data-rest</artifactId>
+                        <groupId>org.springframework.boot</groupId>
+                </dependency>
+
+                <dependency>
+                        <artifactId>spring-boot-starter-test</artifactId>
+                        <groupId>org.springframework.boot</groupId>
+                        <scope>test</scope>
+                </dependency>
+
+                <!--New -->
+                <dependency>
+                        <artifactId>mysql-connector-java</artifactId>
+                        <groupId>mysql</groupId>
+                </dependency>
+
+                <dependency>
+                        <artifactId>spring-boot-starter-data-jpa</artifactId>
+                        <groupId>org.springframework.boot</groupId>
+                </dependency>
+
+                <!-- https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-devtools -->
+                <dependency>
+                        <artifactId>spring-boot-devtools</artifactId>
+                        <groupId>org.springframework.boot</groupId>
+                </dependency>
+
+                <dependency>
+                        <artifactId>lombok</artifactId>
+                        <groupId>org.projectlombok</groupId>
+                        <optional>true</optional>
+                </dependency>
+
+                <dependency>
+                        <groupId>org.springframework</groupId>
+                        <artifactId>spring-test</artifactId>
+                        <version>5.1.8.RELEASE</version>
+                        <scope>compile</scope>
+                </dependency>
+
+                <!-- https://mvnrepository.com/artifact/org.modelmapper/modelmapper -->
+                <dependency>
```

Comment:
Do we need this ?

Reply:
this used for converting dto to enity,
its a mapper.

Reply:
See if we can use [Object Mapper](https://fasterxml.github.io/jackson-databind/javadoc/2.2.0/com/fasterxml/jackson/databind/ObjectMapper.html)